BackendTaskPilot.sln

```diff
@@ -0,0 +1,16 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "BackendTaskPilot", "BackendTaskPilot\BackendTaskPilot.csproj", "{85AE9448-BC75-4412-8D41-DA6C52B676D9}"
```

Comment:
#### Project Structure, Naming, and Separation
This implementation has everything in a single project. It's good practise to separate different concerns into their own projects. 

- Separate logic into different projects so it's easier to maintain and extend
- Place all projects in a `src` folder and all test projects in a `test` folder. 
- Give the solution and projects names that describe the domain. 
- Follow same structure in our other repositories or below structure if no repos exist
- Learn more using links below
  - [Common web application architectures](https://learn.microsoft.com/en-us/dotnet/architecture/modern-web-apps-azure/common-web-application-architectures)
  - [C# at Google Style Guide](https://google.github.io/styleguide/csharp-style.html)
  - [eShop](https://github.com/dotnet/eShop) 

Example:

ExchangeRate - solution name
 - src
   - ExchangeRate.Api
   - ExchangeRate.Application
 - test
   - ExchangeRate.Api.UnitTest 
   - ExchangeRate.Api.IntegrationTest 
   - ExchangeRate.Application.UnitTest